react/week3/todolist/src/components/renderItems.js

```diff
@@ -0,0 +1,90 @@
+import {useState, useEffect} from "react";
+import PropTypes from "prop-types";
+import TodoList from "./Todolist";
+
+function RenderTodos() {
```

Comment:
We usually name components that return JSX elements with a noun. Eg. `Todos`
If the return type was a function or some sort of logic(eg. a state object), then you could name it as renderTodos.